packages/table/README.md

```diff
@@ -0,0 +1,19 @@
+# Table
+
+## Introduction
+
+The Table component gets used to render Tables with the help of a virtual list. It exposes two properties one of them is the `data` which awaits an Array.
+
+The other one is `children` which has to be either an Array of Columns or one Column.
+
+# Columns
```

Comment:
There's no `Columns` (plural) component is there?

Reply:
No you are right.